VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/unread-messages/unread-messages.component.ts

```diff
@@ -82,6 +82,20 @@ export class UnreadMessagesComponent extends UnreadMessagesComponentBase impleme
     handleImReceived(message: InstantMessage) {
         if (this.getHearing().id === message.conferenceId) {
             this.incrementUnreadCounter(message.conferenceId, message.from);
+
+            const messageCount = this.unreadMessages.find(x => x.participant_id === message.from);
```

Comment:
This seems to be a confusing name as the number of unread messages is actually a property inside this. I know it's used elsewhere in the class but we should come up with something more clear.

Reply:
The var has been renamed for logging purposses